python/graphscope/gsctl/commands/dev_command.py

```diff
@@ -0,0 +1,366 @@
+@click.argument(
+    "type",
+    type=click.Choice(
+        ["dev", "client"],
+        case_sensitive=False,
+    ),
+    required=True,
+)
+@click.option(
+    "--repo-home",
+    envvar="REPO_HOME",
+    type=click.Path(),
+    default=os.path.abspath("."),
+    show_default=True,
+    help="GraphScope code repo location.",
+)
+@click.option(
+    "--cn",
+    is_flag=True,
+    default=False,
+    help="Whether to use CN located mirrors to speed up download.",
+)
+@click.option(
+    "--install-prefix",
+    type=click.Path(),
+    default="/opt/graphscope",
+    show_default=True,
+    help="Install built binaries to customized location.",
+)
+@click.option(
+    "--from-local",
+    type=click.Path(),
+    default="/tmp/gs-local-deps",
+    show_default=True,
+    help="""Find raw dependencies of GraphScope from a local directory. The raw
+    dependencies would then be built and installed to [prefix]. If the directory
+    is empty or not exists, dependency files would be downloaded to [directory].""",
+)
+@click.option(
+    "--v6d-version",
+    default="main",
+    show_default=True,
+    help="v6d version to clone.",
+)
+@click.option(
+    "-j",
+    "--jobs",
+    default="2",
+    help="Concurrent jobs in building, i.e., -j argument passed to make.",
+)
+@click.option(
+    "--for-analytical",
+    is_flag=True,
+    default=False,
+    help="Only install analytical engine dependencies.",
+)
+@click.option(
+    "--no-v6d",
+    is_flag=True,
+    default=False,
+    help="Do not install v6d, for build base docker images, could only be used with '--for-analytical'",
+)
+def install_deps(
+    type,
+    repo_home,
+    cn,
+    install_prefix,
+    from_local,
+    v6d_version,
+    jobs,
+    for_analytical,
+    no_v6d,
+):
+    """Install dependencies for building GraphScope."""
+    cmd = [
+        "bash",
+        install_deps_script,
+        "-t",
+        type,
+        "-i",
+        install_prefix,
+        "-d",
+        from_local,
+        "-v",
+        str(v6d_version),
+        "-j",
+        str(jobs),
+    ]
+    if for_analytical:
+        cmd.append("--for-analytical")
+    if no_v6d:
+        if not for_analytical:
+            # could only be used with '--for-analytical'
+            raise RuntimeError("Missing --for-analytical with --no-v6d parameter")
+        cmd.append("--no-v6d")
+    if cn:
+        cmd.append("--cn")
+    run_shell_cmd(cmd, repo_home)
+
+
+@click.command()
+@click.argument(
+    "component",
+    type=click.Choice(
+        [
+            "interactive",
+            "interactive-install",
+            "analytical",
+            "analytical-java-install",
+            "analytical-install",
+            "learning",
+            "learning-install",
+            "coordinator",
+            "client",
+            "clean",
+            "all",
+        ],
+        case_sensitive=False,
+    ),
+    required=False,
+)
+@click.option(
+    "--repo-home",
+    envvar="REPO_HOME",
```

Comment:
Would `GRAPHSCOPE_REPO` or `GRAPHSCOPE_REPOSITORY` be better?

Reply:
Change to `GRAPHSCOPE_REPO ` and `--graphscope-repo`